source/agora/node/Registry.d

```diff
@@ -193,7 +193,7 @@ public class NameRegistry: NameRegistryAPI
 
     ***************************************************************************/
 
-    public Address[] getValidatorsAddresses ()
+    public auto validatorsAddresses ()
```

Comment:
Is this really necessary? They can all be reached at `pubkey.validators.realm` anyways

Reply:
The change or the method itself ?